### ISSUE TITLE
Project Duplicate Filter

### ISSUE THREAD

**Issue by jordanlesich** [DAOmasons/grant-ships-app#176]
### User Stories
- As a game facilitator, I would like to remove duplicate projects from the interface
- As a project member or admin I would like to be able to request that duplicate projects or projects that were configured incorrectly be removed from the app

### Technical Implemtation (Short term)
- Because the project profiles are generated on the Allo side, and we source from Allo, I don't have any way to actually remove these profiles on contract. 
- However, a good short term solution would be to add a filter client side. 
- Filter in the query

### Technical Implemtation (Mid term, create another ticket once ST solution is complete)
- Can use the indexer to remove Projects through the UI, instead of manually hardcoding a hide-list. 
- Create tab in the Facilitator dashboard that accepts a project ID 
- It checks to ensure that the ID exists
- When the Facilitator hits submit, it sends a postUpdate function with a special tag
- When the indexer reads that tag, it locates the project and sets a new ``hidden`` field on the project entity as true. 
- The Project and MyProjects query will then filter out anything with hidden = true
- Using the same pattern, we could also create a way for project members to hide their own profiles. Though due to how the subgraph and postUpdate is currently built, this would be much tougher. Better to do this when we rebuild the indexer. 
- Should also be able to reveal hidden projects?
